migrator/migrations/m_96_to_m_97_modify_default_vulnreportcreator_role/migration_test.go

```diff
@@ -0,0 +1,102 @@
+package m96tom97
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations/rocksdbmigration"
+	"github.com/stackrox/rox/migrator/rockshelper"
+	"github.com/stackrox/rox/pkg/auth/permissions"
+	permissionsUtils "github.com/stackrox/rox/pkg/auth/permissions/utils"
+	rolePkg "github.com/stackrox/rox/pkg/auth/role"
+	"github.com/stackrox/rox/pkg/auth/role/resources"
+	"github.com/stackrox/rox/pkg/rocksdb"
+	"github.com/stackrox/rox/pkg/testutils/rocksdbtest"
+	"github.com/stretchr/testify/suite"
+	"github.com/tecbot/gorocksdb"
+)
+
+func TestMigration(t *testing.T) {
+	suite.Run(t, new(vulnReporterRoleUpdateTestSuite))
+}
+
+type vulnReporterRoleUpdateTestSuite struct {
+	suite.Suite
+
+	rocksDB *rocksdb.RocksDB
+	db      *gorocksdb.DB
+}
+
+var _ suite.TearDownTestSuite = (*vulnReporterRoleUpdateTestSuite)(nil)
+
+func (suite *vulnReporterRoleUpdateTestSuite) SetupTest() {
+	var err error
+	suite.rocksDB, err = rocksdb.NewTemp(suite.T().Name())
+	suite.NoError(err)
+
+	suite.db = suite.rocksDB.DB
+}
+
+func (suite *vulnReporterRoleUpdateTestSuite) TearDownTest() {
+	rocksdbtest.TearDownRocksDB(suite.rocksDB)
+}
+
+func (suite *vulnReporterRoleUpdateTestSuite) TestRolesUpdate() {
```

Comment:
Add a test that an unrelated role/permissionset doesn't get updated

Reply:
Added